lib/fortitude/widget/caching.rb

```diff
@@ -0,0 +1,38 @@
+module Fortitude
+  class Widget
+    module Caching
+      extend ActiveSupport::Concern
+      include ActionView::Helpers::CacheHelper
+
+      module ClassMethods
+        # PUBLIC API
+        def cacheable(opts = {})
+          if extra_assigns == :use
+            extra_assigns :ignore
+          end
+
+          static_keys = Array(opts.fetch(:key, []))
```

Comment:
This allows for doing something like:

```rb
class MyWidget
  cacheable key: 'v2'
end
```

Although heck, if the template is going to be digested, then we could just as well remove this code and instruct this usage:

```rb
class MyWidget
  cacheable # v2
end
```

I guess the only reason this code would be necessary is when using [the `:skip_digest` option](http://api.rubyonrails.org/classes/ActionView/Helpers/CacheHelper.html#method-i-cache).